docs/includes/no-mobile-with-json-note.md

```diff
@@ -1,2 +0,0 @@
-> [!NOTE]
-> Add-ins using the [Unified manifest for Microsoft 365 (preview)](../develop/unified-manifest-overview.md) aren't currently supported on mobile devices.
```

Comment:
Apparently, there are several articles that reference this include. You should remove/replace those references.

Reply:
> Apparently, there are several articles that reference this include. You should remove/replace those references.

That is being done in another PR. We'll merge that one before this one and these errors will go away.